src/html_support_files/odoc.css

```diff
@@ -4,9 +4,105 @@
    %%NAME%% %%VERSION%% */
 
 /* Fonts */
-@import url('https://fonts.googleapis.com/css?family=Fira+Mono:400,500');
-@import url('https://fonts.googleapis.com/css?family=Noticia+Text:400,400i,700');
-@import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,400i,500,500i,600,600i,700,700i');
+/* noticia-text-regular - latin */
+@font-face {
+  font-display: swap; /* Check https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display for other options. */
```

Comment:
I don't think we should configure this. The default is probably the best as we generate a static website with nothing dynamic.

Reply:
Fair enough - I don't have a strong opinion and was just using the default from google-webfonts-helper. Using the default seems reasonable to me, so I'll push a fix. Thanks!